storage/map_files_downloader.hpp

```diff
@@ -15,16 +15,20 @@
 #include <utility>
 #include <vector>
 
+#include "platform/servers_list.hpp"
+
 namespace storage
 {
+using downloader::MetaConfig;
+
 // This interface encapsulates HTTP routines for receiving servers
 // URLs and downloading a single map file.
 class MapFilesDownloader
 {
 public:
   // Denotes bytes downloaded and total number of bytes.
   using ServersList = std::vector<std::string>;
```

Comment:
I kept just `std::vector<std::string>` in downloader class instead of storing entire MetaConfig. The rest of MetaConfig is stored in settings. Therefore, this typedef is needed for m_serversList member.